themes/thecyberden.omp.json

```diff
@@ -0,0 +1,128 @@
+{
+    "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+    "final_space": false,
+    "console_title": false,
+    "console_title_style": "folder",
+    "blocks": [
+        {
+            "type": "prompt",
+            "alignment": "left",
+            "horizontal_offset": 0,
+            "vertical_offset": 0,
+            "segments": [
+                {
+                    "type": "session",
+                    "style": "diamond",
+                    "powerline_symbol": "\uE0B0",
+                    "foreground": "#ffffff",
+                    "background": "#2e9599",
+                    "leading_diamond": "\uE0B6",
+                    "trailing_diamond": "",
+                    "properties": {
+                        "prefix": "\uFCBE ",
+                        "template": "{{ .ComputerName }}"
+                    }
+                },
+                {
+                    "type": "path",
+                    "style": "powerline",
+                    "powerline_symbol": "\uE0B0",
+                    "invert_powerline": false,
+                    "foreground": "#ffffff",
+                    "background": "#0080ff",
+                    "leading_diamond": "",
+                    "trailing_diamond": "\uE0B0",
+                    "properties": {
+                        "prefix": " \uE5FF ",
+                        "style": "full",
+                        "enable_hyperlink": true
+                    }
+                },
+                {
+                    "type": "git",
+                    "style": "powerline",
+                    "powerline_symbol": "\uE0B0",
+                    "invert_powerline": false,
+                    "foreground": "#193549",
+                    "background": "#c19c00",
+                    "leading_diamond": "",
+                    "trailing_diamond": "",
+                    "properties": {
+                        "display_status": true,
```

Comment:
needs to be renamed to `fetch_status`, please validate the other properties as well (and add a `template`)

Reply:
Renamed and added template:
                        "fetch_status": true,
                        "fetch_stash_count": true,
                        "fetch_upstream_icon": true,
                        "template": "{{ .UpstreamIcon }}{{ .HEAD }}{{ .BranchStatus }}{{ if .Working.Changed }} \uF044 {{ .Working.String }}{{ end }}{{ if and (.Working.Changed) (.Staging.Changed) }} |{{ end }}{{ if .Staging.Changed }} \uF046 {{ .Staging.String }}{{ end }}{{ if gt .StashCount 0 }} \uF692 {{ .StashCount }}{{ end }}"